apps/server/src/graphql/resolvers.ts

```
@@ -0,0 +1,1239 @@
+
+        logger.info(`paginatedTokenRecords: Querying ${startDateStr} to ${endDateStr}`);
+
+        const results = await queryAllSubgraphsWithPerChainVariables<SingleChainTokenRecordsResponse>(
+          TOKEN_RECORDS_DATE_RANGE,
+          () => ({ startDate: startDateStr, endDate: endDateStr, pageSize }),
+          logger
+        );
+
+        logger.info(`paginatedTokenRecords: Got results from ${results.successfulChains.length} chains, failed: ${results.failedChains.join(', ')}`);
+
+        results.successfulChains.forEach(c => successfulChainsSet.add(c));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)